### ISSUE TITLE
Android 10 not working on floating.enable()

### ISSUE THREAD

**Issue by euferreira** [wrbl606/floating#24]
Hello everyone, after a lot of validation, I found a possible bugfix in the `cancelAutoEnable` method:
When trying to create validation manually using the ` enable(EnableArguments arguments)` method, a call is made first to cancel which performs a check at the android api level, however, android 10 does not meet the condition, leaving async stuck. 
I only added a return condition to this validation returning false. I would like to know what the procedures are for opening the pull request with the correction.

## Steps to reproduce:
- Create a controller to call Floating() on android 10;
- Call async enable() method and pass const ImmediatePiP() on params;

## My code:
```dart
_floating ??= Floating();

      final pipCurrentStatus = await _floating!.pipStatus;
      final availabled = await _floating!.isPipAvailable;

      if (pipCurrentStatus == PiPStatus.disabled && availabled) {
        await _floating!.enable(const ImmediatePiP());
        _listenPip();
      }
```

**Comment by wrbl606** [comment]
Thanks for reporting! The bug then is on the native Android side, I'll take a look.

**Comment by euferreira** [comment]
I'm going to open a PR with the correction, can you analyze it?

**Comment by euferreira** [comment]
Forget it, now that I see that you've updated it, thank you very much

**Comment by wrbl606** [comment]
Yup, fixed in 4.0.1, thanks again for the report
